kickoff/static/model/release.js

```diff
@@ -1,37 +1,79 @@
+// Please also update VALID_VERSION_PATTERN in forms.py
+// Changing the bracket orders may require to update INDEXES and TYPE_INDEXES
+// TODO: Read the pattern from a centralized file
+var VALID_VERSION_PATTERN_STRING  = '^(\\d+)\\.(' + // Major version number
```

Comment:
I won't score any goal on PES until we read the regex from a single file 😃 

Another reason I think it'd be better to do it in a follow up is that we may need to replace characters depending on the language (for instance `\\` instead of `\` in JS). It will very likely add complexity to the PR.